build-a-react-native-application-for-ios-and-android-from-start-to-finish/notes/07-write-to-the-console-log-in-a-react-native-app.md

```diff
@@ -0,0 +1,18 @@
+# Write to the Console Log in a React Native App
+
+**[📹 Video](https://egghead.io/lessons/react-native-write-to-the-console-log-in-a-react-native-app)**
+
+## Developer tools
+
+To enable the developer tools, first bring up the React Native Developer Menu by shaking the physical device, or pressing `cmd + d` or `cmd + m` on macOS or `ctrl + m` on Windows. Select `Debug JS Remotely` from this menu. This will open a browser window with the full suite of developer tools to help debug our apps.
```

Comment:
```suggestion
⌨️  To enable the developer tools, first bring up the React Native Developer Menu by shaking the physical device, or pressing `cmd + d` or `cmd + m` on macOS or `ctrl + m` on Windows. Select `Debug JS Remotely` from this menu. This will open a browser window with the full suite of developer tools to help debug our apps.
```